test/fixtures/workflow_executions.yml

```diff
@@ -327,6 +344,7 @@ workflow_execution<%= (n) %>:
   workflow_url: "https://github.com/phac-nml/irida_next_example_new"
   run_id: <%= "my_run_id_#{n}" %>
   submitter_id: <%= ActiveRecord::FixtureSet.identify(:jane_doe, :uuid) %>
+  namespace_id: <%= ActiveRecord::FixtureSet.identify(:project1_namespace, :uuid) %>
```

Comment:
Update the `project1_namespace` to `project33_namespace` as that belongs to the user `jane doe`

Reply:
Thanks @deepsidhu85 updated in d61c11eadcfa6a3f20b077ec27cb074ebdeb324b